.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  env: {
+    jest: true,
+    webextensions: true,
+  },
+  parser: "babel-eslint",
+  extends: [
+    "eslint:recommended",
+    "plugin:prettier/recommended",
+    "plugin:mozilla/recommended",
+    "plugin:react/recommended",
+  ],
+  plugins: [
+    "mozilla",
+  ],
+  rules: {
+    "mozilla/no-define-cc-etc": "off",
+    "prefer-const": "error",
+    "lines-between-class-members": ["error", "always"],
+    "padding-line-between-statements": [
+      "error",
+			{
+				blankLine: "always",
+				prev: "multiline-block-like",
+				next: "*",
+			},
```

Comment:
![RewardingFrigidIrishwaterspaniel-size_restricted](https://user-images.githubusercontent.com/119884/79802181-88cb1900-832d-11ea-9cd9-f80b9702306b.gif)

Reply:
thats what i get for copy pasting. guess i need a no-tabs eslint rule too!